TestClientWatchWithRevision is unstable

## Flaky Test

### Which jobs are failing

TestGlobalConfigTestSuite 
### CI link


### Reason for failure (if possible)
```
 --- FAIL: TestGlobalConfigTestSuite (5.14s)
2023-02-17T03:13:58.0444036Z     --- FAIL: TestGlobalConfigTestSuite/TestClientWatchWithRevision (1.06s)
2023-02-17T03:13:58.0444283Z         global_config_test.go:293: 
2023-02-17T03:13:58.0444834Z             	Error Trace:	/home/runner/work/pd/pd/tests/client/global_config_test.go:293
2023-02-17T03:13:58.0445131Z             	Error:      	Not equal: 
2023-02-17T03:13:58.0445435Z             	            	expected: 28
2023-02-17T03:13:58.0445729Z             	            	actual  : 29
2023-02-17T03:13:58.0446939Z             	Test:       	TestGlobalConfigTestSuite/TestClientWatchWithRevision
2023-02-17T03:13:58.0447022Z FAIL
2023-02-17T03:13:58.0447545Z [2023/02/17 03:13:56.633 +00:00] [WARN] [grpclog.go:60] ["grpc: addrConn.createTransport failed to connect to {127.0.0.1:46139  <nil> 0 <nil>}. Err :connection error: desc = \"transport: Error while dialing dial tcp 127.0.0.1:46139: connect: connection refused\". Reconnecting..."]
```

### Anything else

--- a/tests/client/global_config_test.go
+++ b/tests/client/global_config_test.go
@@ -290,7 +290,7 @@ func (suite *globalConfigTestSuite) TestClientWatchWithRevision() {
 	res, revision, err := suite.client.LoadGlobalConfig(suite.server.Context(), nil, globalConfigPath)
 	suite.NoError(err)
 	suite.Len(res, 1)
-	suite.Equal(r.Header.GetRevision(), revision)
+	suite.LessOrEqual(r.Header.GetRevision(), revision)
 	suite.Equal(pd.GlobalConfigItem{EventType: pdpb.EventType_PUT, Name: suite.GetEtcdPath("test"), PayLoad: []byte("test"), Value: "test"}, res[0])
 	// Mock when start watcher there are existed some keys, will load firstly
 	for i := 0; i < 6; i++ {

